CMakeLists.txt

```diff
@@ -45,10 +45,12 @@ IF(NOT DEAL_II_WITH_TRILINOS OR NOT DEAL_II_WITH_P4EST)
   ENDIF()
```

Comment:
Do you want to remove the `FATAL_ERROR` also in the check `DEAL_II_WITH_TRILINOS`?

Reply:
Indeed, I wanted to list all missing dependencies for https://github.com/dealii/dealii/issues/14637. However, once I notices that Trilinos is the problem and there was a merged Trilinos PR in deal.II in the night, I stopped^^